secscanner2junit/sast.py

```diff
@@ -63,8 +65,14 @@ def parse_findings(self, finding, time):
             pass
 
         f_type = finding['identifiers'][0]['name']
+        
+        try:
+            finding_id = properties['id']
+        except KeyError:
+            finding_id = str(random.randrange(1, 10000000))
+        
         if properties['name']:
-            tc = TestCase(name=properties['name'] + " " + properties['id'] , classname=self.p_type, file=properties['file'], elapsed_sec=time, line=properties['start_line'])
+            tc = TestCase(name=properties['name'] + " " + finding_id , classname=self.p_type, file=properties['file'], elapsed_sec=time, line=properties['start_line'])
         else:
             tc = TestCase(name=f_type, classname=self.p_type, file=properties['file'], elapsed_sec=time, line=properties['start_line'])
```

Comment:
- [ ] Since `f_type` is non unique (remember `f_type = finding['identifiers'][0]['name']`), we also need the `finding_id` attached to it

Reply:
This one is the last one missing - afterwards, we can merge :) Thank you for you patience!

Reply:
I think it is now ok, I promie next time i will use gitpod or some IDE instead modify it in WEB xD Really thank you for your super support :)